build/docker/gaia-taskenv/Dockerfile

```diff
@@ -1,14 +1,7 @@
-from taskcluster/tester:0.3.6
+from taskcluster/tester:0.4.4
```

Comment:
So I wasn't aware of gaia-taskenv inheriting from one of our images defined in the gecko tree.  We should talk soon about some changes coming that would impact this.

We are going to be automatically building images and producing them as artifacts of a task instead of utilizing docker hub.  This of course will break inheriting from that image here.